examples/cacoh.py

```diff
@@ -0,0 +1,496 @@
+    data, method="cacoh", indices=multivar_indices, sfreq=100, fmin=3, fmax=35
+)
+
+###############################################################################
+# As you can see below, using CaCoh we have summarised the most relevant
+# connectivity information from our 10 seed channels and 6 target channels as a
+# single spectrum of connectivity values. This lower-dimensional representation
+# of signal interactions is much more interpretable when analysing connectivity
+# in complex systems such as the brain.
+
+# %%
+
+print(f"Results shape: {cacoh.get_data().shape} (connections x frequencies)")
+
+# Plot CaCoh
+fig, axis = plt.subplots(1, 1)
+axis.plot(cacoh.freqs, np.abs(cacoh.get_data()[0]), linewidth=2)
+axis.set_xlabel("Frequency (Hz)")
+axis.set_ylabel("Connectivity (A.U.)")
+fig.suptitle("CaCoh")
+
+###############################################################################
+# Note that we plot the absolute values of the results (coherence) rather than
+# the complex values (coherency). The absolute value of connectivity will
+# generally be of most interest, however information such as the phase of
+# interaction can only be extracted from the complex-valued results, e.g. with
+# the :func:`numpy.angle` function.
+
+# %%
+
+# Plot phase of connectivity
+fig, axis = plt.subplots(1, 1)
+axis.plot(cacoh.freqs, np.angle(cacoh.get_data()[0]), linewidth=2)
+axis.set_xlabel("Frequency (Hz)")
+axis.set_ylabel("Phase of connectivity (radians)")
+fig.suptitle("CaCoh")
+
+###############################################################################
+# CaCoh versus coherence
+# ----------------------
+#
+# To further demonstrate the signal-to-noise ratio benefits of CaCoh, below we
+# compute connectivity between each seed and target using bivariate coherence.
+# With our 10 seeds and 6 targets, this gives us a total of 60 unique
+# connections which is very difficult to interpret without aggregating some
+# information. A common approach is to simply average across these connections,
+# which we do below.
+
+# %%
+
+# Define bivariate connectivity indices
+bivar_indices = seed_target_indices(seeds, targets)
+
+# Compute bivariate coherence
+coh = spectral_connectivity_epochs(
+    data, method="coh", indices=bivar_indices, sfreq=100, fmin=3, fmax=35
+)
+
+###############################################################################
+# Plotting the bivariate and multivariate results together, we can see that
+# coherence still captures the interactions at 10-12 Hz and 23-25 Hz, however
+# the scale of the connectivity is much smaller. This reflects the fact that
+# CaCoh is able to capture the relevant components of interactions between
+# multiple signals, regardless of whether they are present in all channels.
+
+# %%
+
+print(f"Results shape: {coh.get_data().shape} (connections x frequencies)")
+
+cacoh_min = np.min(np.abs(cacoh.get_data()[0]))
+coh_min = np.min(np.mean(coh.get_data(), axis=0))

```

Comment:
Maybe move this to the above section since you mention you are "taking the average" in the above. 

This section seems dedicated to plotting.

Reply:
Moved this to above section an restructured related things.